components/FlashcardBack/index.js

```diff
@@ -25,11 +28,7 @@ function FlashcardBack({ entry, studyList, setIsFront, databaseMutate }) {
             handleWrongAnswer({ studyList, setIsFront, entry, databaseMutate })
           }
         >
-          <span
-            className="inherit-background-color"
-            role="img"
-            aria-label="did not know"
-          >
+          <span role="img" aria-label="did not know">
```

Comment:
Then the screen-reader reads: "Did not know"??

Reply:
Yeah, probably not too intuitive. I changed it